common/test/acceptance/tests/video/test_video_license.py

```diff
@@ -0,0 +1,85 @@
+# coding: utf-8
+from __future__ import unicode_literals
+from nose.plugins.attrib import attr
+from ..studio.base_studio_test import StudioCourseTest
+
+#from ..helpers import UniqueCourseTest
+from ...pages.studio.overview import CourseOutlinePage
+from ...pages.lms.courseware import CoursewarePage
+from ...fixtures.course import XBlockFixtureDesc
+
+
+@attr('shard_1')
+class VideoLicenseTest(StudioCourseTest):
+
+    def setUp(self):
+        super(VideoLicenseTest, self).setUp()
+
+        self.lms_courseware = CoursewarePage(
+            self.browser,
+            self.course_id,
+        )
+        self.studio_course_outline = CourseOutlinePage(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run']
+        )
+
+    # used by StudioCourseTest.setUp()
+    def populate_course_fixture(self, course_fixture):
+        video_block = XBlockFixtureDesc('video', "Test Video")
+        vertical = XBlockFixtureDesc('vertical', "Test Vertical")
+        vertical.add_children(video_block)
+        sequential = XBlockFixtureDesc('sequential', "Test Section")
+        sequential.add_children(vertical)
+        chapter = XBlockFixtureDesc('chapter', "Test Chapter")
+        chapter.add_children(sequential)
+        self.course_fixture.add_children(chapter)
+
+    def test_empty_license(self):
+        self.lms_courseware.visit()
+        video = self.lms_courseware.q(css=".vert .xblock .video")
```

Comment:
@singingwolfboy and I discussed this one too. In these instances, the code is not testing a generic aspect of the page, but an arbitrary component that has been included on it (in this case, video). It probably doesn't make sense to have every possible component query added to the page object, so we agreed that inline queries make sense here.